src/main/java/lotto/domain/WinningNumber.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WinningNumber {
+    private final List<Integer> winningNumbers;
+
+    public WinningNumber() {
+        winningNumbers = new ArrayList<>();
+    }
+
+    public WinningNumber(List<Integer> winningNumber) {
+        this.winningNumbers = winningNumber;
+    }
+
+    public List<Integer> getWinningNumbers() {
+        return winningNumbers;
+    }
```

Comment:
getter를 사용하는 곳이 결국 필요한게 stream인것 같아요. 일급컬렉션이 마치 진짜 컬렉션이 것처럼 스트림을 메서드로 바로 제공하는건 어떨까요?
```suggestion
    public Stream<Integer> stream() {
        return winningNumbers.stream();
    }
```